lib/src/reactive_ble.dart

```diff
@@ -220,14 +220,22 @@ class FlutterReactiveBle {
         .then((message) => message.result.dematerialize());
   }
 
-  /// Scan for devices that are advertising a specific service
-  /// [scanMode] is used only on Android to enforce a more battery or latency intensive scan strategy
+  /// Scan for devices that are advertising a specific services. Use [withServices] filter on devices that are advertising
+  /// the defined services.
   ///
-  Stream<DiscoveredDevice> scanForDevices(
-      {@required Uuid withService, ScanMode scanMode = ScanMode.balanced}) {
+  /// There are two Android specific parameters that will be ignored on iOS:
+  /// [scanMode] allows to choose between different levels of power efficient and/or low latency scan modes.
+  /// [requireLocationServicesEnabled] is used to enforce location services to be enabled while scanning. If this parameter set to
```

Comment:
If you want to break this text into multiple paragraphs, you need to add empty lines to indicate that (are you familiar with Markdown?). If you want to make it a list, you need some bullets (hyphens or asterisks).